src/Raygun4php/RaygunClient.php

```diff
@@ -361,7 +361,7 @@ private function post($data_to_send, $cert_path)
         if ($this->proxy) {
           $curlOpts[] = "--proxy '" . $this->proxy . "'";
         }
-        $cmd = "curl " . implode(' ', $curlOpts) . " 'https://api.raygun.io:443/entries' > /dev/null 2>&1 &";
+        $cmd = "curl " . implode(' ', $curlOpts) . " 'https://$this->host:$this->port$this->path' > /dev/null 2>&1 &";
```

Comment:
```suggestion
        $cmd = "curl " . implode(' ', $curlOpts) . " 'https://{$this->host}:{$this->port}{$this->path}' > /dev/null 2>&1 &";
```

Reply:
Thank you for the quick feedback!
The PR is now updated with your suggested improvements.